### DISCUSSION TITLE
(Fortaleza - CE) Desenvolvedor C# JR / Pleno / Sênior - @ExpressoGuanabara

### DISCUSSION THREAD

**Discussion by lucasapoena** [DevComputaria/Forum#5] [Vagas]
![VagaDesenvolvedor1080x1080-2](https://user-images.githubusercontent.com/135553/106811405-0bb72900-664d-11eb-831c-d5aa2229ec64.png)

**Comment by marcialwushu** [comment]
Vamos criar um repositório para compartilhar vagas de TI 
Assim centralizamos a demanda desse tipo de informação nesse repositório 

O que acham ???

**Reply by lucasapoena** [reply]
Pode ser, acho que pode ficar mais organizado pelos issues

**Reply by marcialwushu** [reply]
https://github.com/DevComputaria/CangaceiroDevVagas/issues
